src/hipoc/hipoc_program.cpp

```diff
@@ -328,8 +328,7 @@ void HIPOCProgramImpl::BuildCodeObject(std::string params,
     }
     else if(miopen::EndsWith(filename, ".cl"))
     {
-        params +=
-            " -Werror" + (is_kernel_str ? MiopengemmWarningsString() : OclKernelWarningsString());
```

Comment:
[Recommendation] IIRC this `is_kernel_str` is a clumsy legacy stuff used for miopengemm kernels only. Please check (see `KernelCache::AddKernel()`). It would be nice to get rid of it everywhere in the same PR (tip: look for `is_kernel_str` in all sources).

Reply:
`is_kernel_str` has been removed

Reply:
👍 👍 👍